kong/plugins/aws-lambda/handler.lua

```diff
@@ -282,7 +285,11 @@ function AWSLambdaHandler:access(conf)
 
     status = proxy_response.status_code
     headers = kong.table.merge(headers, proxy_response.headers)
-    content = proxy_response.body
+    if proxy_response.isBase64Encoded then
+      content = ngx_decode_base64(proxy_response.body)
+    else
+      content = proxy_response.body
+    end
```

Comment:
move this logic in `extract_proxy_response`

Reply:
Will do